Makefile

```diff
@@ -9,6 +9,20 @@ junit_xml := junit.xml
 test_log := test.log
 test_target := .
 lint_check := .ci/lint.sh
+m3db_package := github.com/m3db/m3db
+gopath_prefix := $(GOPATH)/src
+vendor_prefix := vendor
+license_dir := $(vendor_prefix)/github.com/uber/uber-licence
+license_node_modules := $(license_dir)/node_modules
+auto_gen := .ci/auto-gen.sh
+mockgen_package := github.com/golang/mock/mockgen
+mocks_output_dir := generated/mocks/mocks
+mocks_rules_dir := generated/mocks
+protoc_go_package := github.com/golang/protobuf/protoc-gen-go
+proto_output_dir := generated/proto/persistfs
```

Comment:
Might need the ability to add more proto packages, etc.  Right now commit logs have their own proto definitions.  Or we could keep it at some high level package with a name other than `persistfs`?